Bottom: 5e1ce90875ccd0e031058cd93179e55e43704bf0
Top:    7411129e6c92aae16c859c12b7fd9fd8999a12c3
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-16 21:20:12 +0300

Refresh of cli.patch

---

diff --git a/ib-sock-int.h b/ib-sock-int.h
index 84b91d6..add0b53 100644
--- a/ib-sock-int.h
+++ b/ib-sock-int.h
@@ -7,8 +7,6 @@
 #include <rdma/ib_verbs.h>
 #include <rdma/rdma_cm.h>
 
-#define IB_CQ_EVENTS_MAX 128
-
 #define IB_ADDR_TIMEOUT 100
 #define IB_ROUTE_TIMEOUT 100
